README.md

```diff
@@ -31,7 +31,7 @@ Add this dependency to your project's build file:
   }
 
   dependencies {
-     implementation "com.github.basis-theory.basistheory-android:lib:master-SNAPSHOT"
+      implementation 'com.github.basis-theory:basistheory-android:eng-3834-SNAPSHOT'
```

Comment:
Was it intentional to change the branch reference to `eng-3834`?

Reply:
No it wasn't! Good catch.